super-size/many-extensions/src/test/java/io/quarkus/ts/many/extensions/ManyExtensionsIT.java

```diff
@@ -1,25 +1,16 @@
 package io.quarkus.ts.many.extensions;
 
+import static io.restassured.RestAssured.given;
 import static org.hamcrest.CoreMatchers.is;
 
 import org.apache.http.HttpStatus;
 import org.junit.jupiter.api.Test;
 
-import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.QuarkusScenario;
-import io.quarkus.test.services.QuarkusApplication;
-import io.restassured.specification.RequestSpecification;
 
 @QuarkusScenario
 public class ManyExtensionsIT {
 
-    @QuarkusApplication
-    static RestService app = new RestService();
-
-    protected RequestSpecification given() {
```

Comment:
What trick is used so that `RestService` app doesn't need to be defined?

Reply:
https://github.com/quarkus-qe/quarkus-test-framework/blob/1e625fde7b6077b6896d7de53767c555f4e7483b/quarkus-test-core/src/main/java/io/quarkus/test/bootstrap/RestService.java#L22

When there is only one application, it configures well the RestAssured default settings.

Reply:
👍